pkg/config/remote/service/service.go

```diff
@@ -292,6 +307,38 @@ func (s *Service) Stop() error {
 	return s.db.Close()
 }
 
+func (s *Service) pollOrgStatus() {
+	response, err := s.api.FetchOrgStatus(context.Background())
+	if err != nil {
+		// Unauthorized and proxy error are caught by the main loop requesting the latest config,
+		// and it limits the error log.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** (‏For my general understanding) so we'd have that error log in addition to the ones generated (and rate-limited) by the main loop, correct?